addons/ai/XEH_unitInit.sqf

```diff
@@ -1,45 +1,35 @@
-#include "script_component.hpp"
-
-params [["_unit", objNull]];
-
-if (isNull _unit || {isPlayer _unit}) exitWith {};
-
-_unit addEventHandler ["Local", FUNC(handleLocality)];
-
-if (!local _unit) exitWith {};
-
-[{
-    [{
-        params ["_unit"];
-
-        [_unit] call FUNC(assignLoadout);
-
-        _unit disableAI "SUPPRESSION";
-        _unit setBehaviour "AWARE";
-        _unit setCombatMode "RED";
-        _unit setSpeedMode "FULL";
-        _unit allowFleeing 0;
```

Comment:
Agreed with removing above, although not sure how much of an impact will it make on the problem of AI being too easy to kill. This feels more like a feature to eliminate stragglers from defeated AI squads that Zeus would have to deal with.